### PR TITLE
Implement new Home page

### PR DESCRIPTION
**Pull request by fabryscript**
### ClickUp Task

[ClickUp Task URL](PASTE_CLICKUP_TASK_URL_HERE)

### CODE REVIEW

**Comment by fabryscript** [comment]
TODO: add translation labels

File: packages/web/components/chart/home-chart.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import { AreaData, MouseEventParams, Time } from "lightweight-charts";
```
Comment (jonator) [inline packages/web/components/chart/home-chart.tsx:1]: Why not visx? We already import it.

File: packages/web/components/home/hooks.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import { TokenHistoricalPrice } from "@osmosis-labs/stores/build/queries-external/token-historical-chart/types";
```
Comment (jonator) [inline packages/web/components/home/hooks.ts:1]: Let's use the type from the server file instead. Be sure to use `import type` to only import the TS type.

**Review by jonator — COMMENTED** [review]
Initial scan

File: packages/web/components/chart/home-chart.tsx (hunk 1 above)
Comment (DavideSegullo) [reply packages/web/components/chart/home-chart.tsx:1]: We had started out using visx, but there are behaviors that it doesn't handle natively, and in the reference indicated they were using lightweight-chart.

I can think whether it is possible to do the same things with visx, but right now the biggest problem is that the handling of asset x is a bit buggy on visx

**Comment by MaxMillington** [comment]
@fabryscript @DavideSegullo Can we close this PR? I think we are starting the new homepage designs from scratch

**Pull request closed without merging**